src/lib/pages/podcast-episodes/ui/specs/podcast-episodes-page.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { render } from '@testing-library/svelte';
+import { MockComponent } from '$lib/shared/lib/jest-hacks';
+import type { Podcast } from '$lib/shared/api';
+import { PodcastEpisodesPage } from '../..';
+
+jest.mock('~icons/ic/round-arrow-back', () => ({ default: MockComponent }), {
+  virtual: true,
+});
+jest.mock('~icons/ic/round-play-arrow', () => ({ default: null }), { virtual: true });
+jest.mock('~icons/ic/round-pause', () => ({ default: null }), { virtual: true });
+jest.mock('~icons/ic/round-replay-10', () => ({ default: null }), { virtual: true });
+jest.mock('~icons/ic/round-forward-30', () => ({ default: null }), { virtual: true });
+jest.mock('~icons/ic/sharp-favorite', () => ({ default: null }), { virtual: true });
+jest.mock('~icons/ic/twotone-favorite', () => ({ default: null }), { virtual: true });
+jest.mock('~icons/ic/twotone-file-download-done', () => ({ default: null }), {
```

Comment:
Is there really no way around this mess? Is it not possible to write `jest.mock('~icons/*')`, or even add them all to one big file that Jest would automatically include in all tests?

Reply:
It's not possible to have wildcards in mocks, but the other suggestion is a really nice idea which I've now implemented